movies-search-engine/download_poster.py

```diff
@@ -0,0 +1,12 @@
+import pandas as pd
+import urllib.request
+data=pd.read_csv("/home/hp/Documents/weaviate-movie-recommendation/final_data.csv")
```

Comment:
We need to update any references to your local drive, like this:
`/home/hp/Documents/weaviate-movie-recommendation/`
make them relative to the project, (i.e. the `weaviate-movie-recommendation` should be a folder in the project) and provide instructions on where to get the images.